internal/models/api/common.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+type ValidationError struct {
+    Fields []string
+    Message string
+}
+
+type Validator interface {
```

Comment:
How about naming this interface as a verb, `Validatable`?

Reply:
I think the convention in Go is to name interfaces with an -er suffix, unlike Java (https://go.dev/doc/effective_go#interface-names).

Reply:
Another instance of golang being better than java at something. :)